elixir/apps/domain/priv/repo/migrations/20250630210514_drop_foreign_key_constraint_on_change_logs.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Domain.Repo.Migrations.DropForeignKeyConstraintOnChangeLogs do
+  use Ecto.Migration
+
+  def up do
+    drop(constraint(:change_logs, :change_logs_account_id_fkey))
```

Comment:
This is needed because when cascading deletes for an account, we can't readily know that the records we're seeing belong to a deleted account.

I don't expect this to be a major issue. When the feature gets closer to shipping we'll need a periodic job to truncate this table which will take care of the deleted account case.